client/packages/system/src/Encounter/DetailView/Toolbar.tsx

```diff
@@ -95,10 +103,16 @@ export const Toolbar: FC<ToolbarProps> = ({ encounter, onChange }) => {
                 }
               />
             </Box>
-            <Box display="flex" gap={1}>
+            <Box display="flex" gap={0.5}>
```

Comment:
Had to make the program text field wider, so have adjusted the gaps to line up the components